lib/sequent/configuration.rb

```diff
@@ -109,6 +112,7 @@ def initialize
       self.strict_check_attributes_on_apply_events = DEFAULT_STRICT_CHECK_ATTRIBUTES_ON_APPLY_EVENTS
 
       self.logger = Logger.new(STDOUT).tap {|l| l.level = Logger::INFO }
+      self.error_locale = DEFAULT_ERROR_LOCALE
```

Comment:
`error_locale_resolver` is perhaps a better name since it needs to resolve the locale.